### PR TITLE
Updated the endor logo

### PR DESCRIPTION
**Pull request by naveensrinivasan**
Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

### CODE REVIEW

**Comment by naveensrinivasan** [comment]
@azeemshaikh38  The logo didn't show up 😞 . Any idea what needs to happen for the logo to show up?

File: scorecards-site/content/home.md [hunk 1]
```diff
@@ -309,7 +309,7 @@ Security Scorecards helps to make informed decisions about security risks and vu
 
 <div>
 
-![endor](assets/logos/endor.svg)
+![endor](assets/logos/endor.png)
```
Comment (azeemshaikh38) [inline scorecards-site/content/home.md:312]: Leave this as `.svg` file. Some internal magic converts the `.png` to `.svg` and puts it here. But it doesn't work if you provide `.svg` directly.

Comment (naveensrinivasan) [reply scorecards-site/content/home.md]: Ok, I tried that looks like it is still not rendering. Anything else am I missing? If not who else can help with this?

Comment (azeemshaikh38) [reply scorecards-site/content/home.md]: I miss guided you. The real solution was to resize the original SVG file to be smaller and move it from `assets/logos` to `static/assets/logos`. Hopefully, this should work now.


>  If not, who else can help with this?

At this point, it's just us (Scorecard maintainers) :) We might need a bit of trial and error to get a hang of this. I'll try to write up documentation for this as I learn stuff.

**Review by azeemshaikh38 — APPROVED** [review]

File: scorecards-site/content/home.md (hunk 1 above)
Comment (azeemshaikh38) [reply scorecards-site/content/home.md]: Looks good to submit now.

**Pull request merged**